packages/front-end/components/PowerCalculation/PowerCalculationSettingsModal.tsx

```diff
@@ -256,17 +313,31 @@ const MetricParamsInput = ({
     <div className="card gsbox mb-3 p-3 mb-2 power-analysis-params">
       <div className="card-title uppercase-title mb-3">{name}</div>
       <div className="row">
-        {Object.keys(params).map(
-          (entry: keyof Omit<MetricParams, "name" | "type">) => (
+        {Object.keys(params)
+          .filter((v) => !(bayesianParams as readonly string[]).includes(v))
+          .map((entry: keyof Omit<MetricParams, "name" | "type">) => (
             <InputField
               key={`${name}-${entry}`}
               entry={entry}
               form={form}
               metricId={metricId}
             />
-          )
-        )}
+          ))}
       </div>
+      {engineType === "bayesian" && (
```

Comment:
IMO these settings belong in a dropdown on the stats engine settings page as overrides you can set up after the fact, or they should be hidden by default with a small button to "modify metric priors" rather than presenting users with 5(!) input boxes per metric. If you modify metric priors it can open the toggle and two prior fields. You can see how we do this in the MetricForm index.tsx file where we only open the metric prior settings when you turn the override on.

Reply:
@romain-growthbook Can we turn the toggle just into a link text that says "Modify metric priors" and make it much smaller? Or at least change the copy to "Modify metric priors" It's still really busy IMO.

Reply:
Text updated. I'm not sure exactly how to have a good UX that's smaller. Text only, I don't see how you materialize open/close state. Maybe something to ask Kari?

Reply:
I like the toggle.  Nit: can it say "modify metric prior"?